### ISSUE TITLE
Rogue backtick in one of the scripts

### ISSUE THREAD

**Issue by youri--** [memstechtips/UnattendedWinstall#41]
Search for "GetM`ethods". I don't think that backtick is supposed to be there.

https://github.com/memstechtips/UnattendedWinstall/blob/1745581102adf57082c38e81bac7650ead9cf062/autounattend.xml#L1424

Thanks!

**Comment by memstechtips** [comment]
You're actually right lol.

That's not my script though, it was made by AveYo and changed by Chris Titus.
I don't think that actually makes a difference in the environment that script runs in the answer file, but I'll keep it in mind to fix it for future updates, thanks!

**Comment by DutchMon** [comment]
To follow up even further, I think the backtick is intentional and its purpose is to avoid being false flagged and deleted by anti virus software. Here's a summary/breakdown of what's happening.

When this script runs, the backtick in `GetM\`ethods` is ignored, and the method executes as if it were written correctly as `GetMethods`. In PowerShell, the backtick is an escape character used to indicate that the following character should be treated literally. Since there's no special character following the backtick in this context, it essentially gets ignored, and `GetM\`ethods` is interpreted as `GetMethods`.

Here's a breakdown of how PowerShell handles it:

1. **Backtick as Escape Character:** The backtick is used in PowerShell to escape characters that have special meaning. For example, you might use it to include a literal newline or quote character in a string. However, in this context, the backtick is followed by a regular letter (`e`), so it doesn't escape anything.

2. **Ignoring the Backtick:** Since there's nothing special to escape, PowerShell treats the backtick followed by `e` as just `e`. Therefore, `GetM\`ethods` is interpreted as `GetMethods`.

3. **Execution:** The line of code executes as if it were written correctly as `GetMethods`.

To further illustrate, here's a simple example:

```powershell
# Define a string with an embedded backtick
$testString = "Hello` World"
Write-Output $testString
```

In this example, the output will be:

```
Hello World
```

The backtick followed by a space is ignored, and the string is output as if the backtick were not there.

So, in the script line:

```powershell
$D1=[uri].module.gettype('System.Diagnostics.Process')."GetM`ethods"(42) |where {$_.Name -eq 'SetPrivilege'}
```

The `GetM\`ethods` is effectively `GetMethods`, and the method call proceeds as expected.

### Key Points:

- **Obfuscation Technique:** The backtick in `GetM\`ethods` is likely an obfuscation technique to bypass simple text-based detection mechanisms.
- **Script Execution:** Despite the backtick, the method call works correctly, and `GetM\`ethods` functions as `GetMethods`.
- **Privileges Adjustment:** The method `SetPrivilege` is accessed correctly, allowing the script to adjust the process's privileges.

In conclusion, when this script runs, the backtick in `GetM\`ethods` does not affect the functionality, and `GetM\`ethods` executes as `GetMethods`.

**Comment by memstechtips** [comment]
That is quite interesting, thanks for sharing!
